### DISCUSSION TITLE
Problem with the working of the wireui site

### DISCUSSION THREAD

**Discussion by MarcinStefaniak** [wireui/wireui#475] [General]
Hi, Do you know when the documentation page will start?
Currently livewire-wireui.com is showing error 520.

**Comment by ph7jack** [comment]
@MarcinStefaniak It's strange, the doc is now working, did you see a Cloudflare error page?

**Reply by MarcinStefaniak** [reply]
@PH7-Jack 
Yes. I took the screenshot just now.

#edit
Weird. On the phone on the same internet connection, the page is loading. Unfortunately, not on a pc.

![image](https://user-images.githubusercontent.com/65134918/189529435-ed48e1af-302e-4e6e-9d1a-bb06ee8c09d6.png)

**Reply by MarcinStefaniak** [reply]
Ok, I have solved the problem of cleaning browsing files and cookies in the browser. Now it started to work.

**Reply by ph7jack** [reply]
Glad to know
